app/src/main/java/com/x8bit/bitwarden/MainViewModel.kt

```diff
@@ -201,6 +203,14 @@ class MainViewModel @Inject constructor(
                     )
             }
 
+            completeRegistrationData != null -> {
+                specialCircumstanceManager.specialCircumstance =
+                    SpecialCircumstance.CompleteRegistration(
+                        completeRegistrationData = completeRegistrationData,
+                        timestamp = System.currentTimeMillis(),
```

Comment:
⛏️ I think the preference has been to use `Clock.millis()` which will get the same thing, but will be easier for testing since you won't need to statically mock it. `Clock` should be able to just be injected.